test/e2e/dashboard.go

```diff
@@ -28,28 +28,28 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("kube-ui", func() {
+var _ = Describe("Kubernetes Dashboard", func() {
 	const (
-		uiServiceName = "kube-ui"
+		uiServiceName = "kubernetes-dashboard"
 		uiAppName     = uiServiceName
 		uiNamespace   = api.NamespaceSystem
 
 		serverStartTimeout = 1 * time.Minute
 	)
 
-	f := NewFramework("kube-ui")
+	f := NewFramework(uiServiceName)
 
-	It("should check that the kube-ui instance is alive [Conformance]", func() {
-		By("Checking the kube-ui service exists.")
+	It("should check that the kubernetes-dashboard instance is alive [Conformance]", func() {
```

Comment:
I feel bad about it.  `v1.1` conformance tests should pass on a `v1.2` cluster (with some very small number of exceptions, treated like an API breaking change).